src/main/tools/travis/deploy

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+#
+# A simple script to publish our artifacts on a push to the "master" branch.
+# Publishes snapshots or, if our version isn't a snapshot, stable artifacts.
+#
+
+[ -z "$DEPLOYMENT_JDK" ] && DEPLOYMENT_JDK="openjdk11"
+
+# Get the directory from which this script is running
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TAG="$("$DIR"/next_version)"
+
+# If we're building a stable release, check that next tag matches requested tag;
+# also, since we could be testing against multiple JDKs, only deploy on OpenJDK 11
+if [[ "$TRAVIS_PULL_REQUEST" == "false"
+    && ("$TAG" == "$TRAVIS_TAG" || "$TRAVIS_BRANCH" == "master" || "$TRAVIS_BRANCH" == "main"
```

Comment:
I think we can remove "master" from this check

Reply:
This file doesn't need to be checked in. It comes from the parent project (which does support some projects whose branches have not yet been renamed to "main").